packages/ui/src/memberships/components/ProfileComponent.tsx

```diff
@@ -18,7 +21,23 @@ export function ProfileComponent() {
       <MemberBalance>
         <BalanceTitle>Total Balance</BalanceTitle>
         <TotalBalance>
-          <TotalTokenValue value={total} />
+          <TokenDetail>
+            <TotalTokenValue value={total} />
+            <StyledTooltip
+              tooltipLinkText="Learn how to earn JOY’s"
+              tooltipLinkURL="https://www.joystream.org/token/"
+              placement="top-start"
+            >
+              <StyledDefaultTooltip>
+                <QuestionIcon />
+              </StyledDefaultTooltip>
+            </StyledTooltip>
+          </TokenDetail>
+          <TextSmall>
+            <BuyTokenLink href="https://www.joystream.org/token" target="_blank">
```

Comment:
Based on this [message](https://discord.com/channels/811216481340751934/1035125811952504882/1148281281575137292) `www.joystream.org` will change the MEXC link to https://www.mexc.com/exchange/JOYSTREAM_USDT?_from=market so this solve the problem I mentioned in the last review. As a result this link should become:
```suggestion
            <BuyTokenLink href="https://www.joystream.org/token/#exchanges" target="_blank">
```